1)The practice of “phishing,” or using the Internet to pilfer personal information, has become an
increasing concern, not only for individual computer users but also for online retailers and financial
institutions. The Wall Street Journal reported 28% of people who bank online have cut back on their
Internet use. The North Central Educators Credit Union instituted an extensive online security and
educational program six months ago in an effort to combat phishing before the problem became
extreme. The credit union’s managers are certain that while Internet use may be down, the rate for their
customers is much less than 28%. However, they believe that if more than 10% of their customers have
cut back on their Internet banking transactions, they will be required to take more stringent action to
lower this percentage. The credit union’s Information Technology department analyzed 200 randomly
selected accounts and determined that 24 indicated they had cut back on their Internet banking
transactions.

a) State the appropriate null and alternative hypotheses for this situation.

Ho : internet usage <= 0.1,
Ha : internet usage > 0.1

b) Using alpha = 0.05 and the p-value approach, indicate whether the sample data support the
managers’ contention.

In [2]:
import scipy.stats as st 
import math as m

In [8]:
p = 0.1
a = 0.05
n = 200
x = 24

z = st.norm.ppf(1-a)

pe = x / n

print (pe)
print(z)

0.12
1.64485362695


In [11]:
value1 = pe - p
value2 = m.sqrt(p * (1 - p) / n)
z = value1 / value2

print (z)

0.9428090415820628


Since z = 0.9428 < 1.645, therefore accept Ho

4)A survey by the Pew Internet & American Life Project found that 21% of workers with an e-mail account
at work say they are getting more spam than a year ago. Suppose a large multinational company, after
implementing a policy to combat spam, asked 198 randomly selected employees with e-mail accounts at
work whether they are receiving more spam today than they did a year ago. The results of the survey
are in the file Spam. At the 0.025 level of significance, can the company conclude that the percentage of
its employees receiving more spam than a year ago is smaller than that found by the Pew study?

Ho : p >= 0.21,
Ha : p < 0.21

In [16]:
import pandas as pd
import numpy as np
import scipy.stats as st 
import math as m

In [84]:
a = 0.025
n = 198
p = 0.21

In [88]:
z = st.norm.ppf(a)
print (z)

-1.95996398454


In [71]:
df = pd.read_excel('D://Business Statistic/MANB1123-master/MANB1123-master/Data_Set/Spam.xlsx', header=None, skiprows=3)
df.head()

0  1    2
0 NaN  1  Yes
1 NaN  2  Yes
2 NaN  3   No
3 NaN  4   No
4 NaN  5   No

In [72]:
spam2 = df.loc[:, "1":"2"]
spam2.head()

1    2
0  1  Yes
1  2  Yes
2  3   No
3  4   No
4  5   No

In [73]:
spam = spam2.rename(columns={1: 'Worker', 2: 'Response: More Spam Today'})
spam.head()

Worker Response: More Spam Today
0       1                       Yes
1       2                       Yes
2       3                        No
3       4                        No
4       5                        No

In [90]:
response = spam[(spam['Response: More Spam Today'] == 'Yes')] 
x= len(response)

In [85]:
#Get point estimate
pe = x / n
pe

0.15151515151515152

In [87]:
value1 = pe - p
value2 = m.sqrt(p * (1 - p) / n)
za = value1 / value2
print (za)

-2.020471697345507


za (-2.02) < z (-1.96), therefore Ho is rejected